src/index.ts

```diff
@@ -23,8 +24,16 @@ const onloadOpt: OnLoadOptions = {
   filter: /\.(hbs|handlebars)$/i,
 };
 
-function hbs(options: { additionalHelpers: any; additionalPartials: any; precompileOptions: any } = { additionalHelpers: {}, additionalPartials: {}, precompileOptions: {} }) {
-  const { additionalHelpers = {}, additionalPartials = {}, precompileOptions = {} } = options;
+type HBSOptions = {
+  additionalHelpers?: any;
+  additionalPartials?: any;
+  precompileOptions?: any;
+  minifyOutput?: boolean;
+  minifyOptions?: any;
+};
+
+function hbs(options: HBSOptions = {}) {
+  const { additionalHelpers = {}, additionalPartials = {}, precompileOptions = {}, minifyOutput = false, minifyOptions = {} } = options;
   return {
     name: "handlebars",
     setup(build: PluginBuild) {
```

Comment:
we can mark this setup function async, then make the import(minifer module) statement